### PR TITLE
Initial Commit --added code for Q1 and Q2

### PR DESCRIPTION
**Pull request by Vaishnavi292000**
Added the code for question 1 and question 2

### CODE REVIEW

File: 2022/GeekGoddess_Final/Question1/solution.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* Read input from STDIN. Print your output to STDOUT*/ 
+#include <bits/stdc++.h>
+using namespace std;
+
+/*
+    SCORE : 100/100
```
Comment (cladius) [inline 2022/GeekGoddess_Final/Question1/solution.cpp:7]: Pls add author info as well.

File: 2022/GeekGoddess_Final/Question2/solution.cpp [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* Read input from STDIN. Print your output to STDOUT*/
```
Comment (cladius) [inline 2022/GeekGoddess_Final/Question2/solution.cpp:1]: Pls mention author and score

**Review by cladius — CHANGES_REQUESTED** [review]

File: 2022/GeekGoddess_Final/Question1/solution.cpp (hunk 1 above)
Comment (Vaishnavi292000) [reply 2022/GeekGoddess_Final/Question1/solution.cpp:7]: Done

File: 2022/GeekGoddess_Final/Question2/solution.cpp (hunk 2 above)
Comment (Vaishnavi292000) [reply 2022/GeekGoddess_Final/Question2/solution.cpp:1]: Done

**Review by cladius — APPROVED** [review]

**Pull request merged**